jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,563 @@
+        "number": 1,
+        "state": "created",
+        "created_at": "2020-01-10T19:45:58.517Z",
+        "trigger": {
+        "received_at": "2020-01-10T19:45:58.489Z",
+          "type": "api",
+                "actor": {
+                  "login": "teesloane",
+                  "avatar_url": "https://avatars0.githubusercontent.com/u/12987958?v=4"
+                }
+              },
+              "vcs": {
+                "origin_repository_url": "https://github.com/{YOUR_USER_NAME}/hello-world",
+                "target_repository_url": "https://github.com/{YOUR_USER_NAME}/hello-world",
+                "revision": "ca67134f650e362133e51a9ffdb8e5ddc7fa53a5",
+                "provider_name": "GitHub",
+                "branch": "master"
+          }
+        }
+        ]
+      }
+    ```
+        
+    That's great! Hopefully everything is working for you up to this point. Let's move on to performing something that might be a bit more useful.
+
+5. One of the benefits of the CircleCI API v2 is the ability to remotely trigger pipelines with parameters. The following code snippet simply triggers a pipeline via `curl` without any body parameters:
+
+    ```sh
+    curl -X POST https://circleci.com/api/v2/project/gh/{YOUR_USER_NAME}/hello-world/pipeline \
+    -H 'Content-Type: application/json' \
+    -H 'Accept: application/json' \
+    -H "Circle-Token: $CIRCLECI_TOKEN" \
+          
+    # Which returns:
+    {
+      "number": 2,
+      "state": "pending",
+      "id": "e411ea74-c64a-4d60-9292-115e782802ed",
+      "created_at": "2020-01-15T15:32:36.605Z"
+    }
+    ```
+
+    While this alone can be useful, we want to be able to customize parameters of the pipeline when we send this POST request. By including a body parameter in the `curl` request (via the `-d` flag), we can customize specific attributes of the pipeline when it runs: pipeline parameters, the branch, or the git tag. Below, we are telling the pipelines to trigger for "my-branch"
+
+    ```sh
+    curl -X POST https://circleci.com/api/v2/project/gh/teesloane/hello-world/pipeline \
+    -H 'Content-Type: application/json' \
+    -H 'Accept: application/json' \
+    -H "Circle-Token: $CIRCLE_TOKEN" \
+    -d '{ "branch": "bar" }' 
+    ```
+
+6. Let's move on to a more complex example: triggering a pipeline and passing a parameter that can be dynamically substituted into your configuration. In this example, we will pass a docker image tag to our docker-executor key. First, we will need to modify the `.circleci/config.yml` to be a little more complex than the standard "Hello World" sample provided by the onboarding.
+
+    ```yaml
+    version: 2.1
+    jobs: 
+      build: 
+        docker: 
+          - 
+            image: "circleci/node:<< pipeline.parameters.image-tag >>"
+        environment: 
+          IMAGETAG: "<< pipeline.parameters.image-tag >>"
+        steps: 
+          - 
+            run: "echo \"Image tag used was ${IMAGETAG}\""
```

Comment:
This should be on the same line as the dash.

Also, instead of escaping quotes, you can do this:

```
- run: "echo 'Image tag used was ${IMAGETAG}'"
```

or not use them on the outside at all:

```
- run: echo "Image tag used was ${IMAGETAG}"
```

Reply:
Fixed! (and thanks for the tips!)